Description: Do not auto-regenerate stuff in the debian/ directory.
---
 Makefile.am  |    3 +--
 configure.ac |    2 --
 2 files changed, 1 insertion(+), 4 deletions(-)

--- mjpegtools.orig/Makefile.am
+++ mjpegtools/Makefile.am
@@ -24,8 +24,7 @@ SUBDIRS = \
 	y4mdenoise \
 	y4mscaler \
 	y4munsharp \
-        y4mutils \
-	debian
+        y4mutils
 
 #
 # Add any non autoconf'd files here, extra readmes and other misc 
--- mjpegtools.orig/configure.ac
+++ mjpegtools/configure.ac
@@ -578,8 +578,6 @@ dnl Output a Makefile or two and the lib
 dnl **********************************************************************
 AC_CONFIG_FILES([
      Makefile
-     debian/Makefile
-     debian/control
      docs/Makefile
      lavtools/Makefile
      yuvcorrect/Makefile
